submissions/AnzhelikaKh/popup_task/style.css

```diff
@@ -0,0 +1,164 @@
+*{
+    font-family: cursive;
+    font-size: 18px;
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    list-style: none;
+    text-decoration: none;
+    font-family: cursive;
+    font-size: 18px;
+    color: rgb(91, 91, 91);
+}
+
+.header{
+    display: flex;
+    align-items: center;
+    height: 90px;
+    border-bottom: 1px solid #000;
+    box-shadow: 0 2px 5px #000;
+
+}
+.logo img{
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css). Also, avoid too specific CSS selectors. [Read about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)